frontend/dockerfile/parser/fuzz.go

```diff
@@ -0,0 +1,27 @@
+package parser
+
+import (
+	"os"
+)
+
+func Fuzz(data []byte) int {
+	f, err := os.Create("Dockerfile")
+	if err != nil {
+		return -1
+	}
+	defer f.Close()
+	defer os.Remove("Dockerfile")
+	_, err = f.Write(data)
+	if err != nil {
+		return -1
+	}
+	df, err := os.Open("Dockerfile")
```

Comment:
No need to retain two FDs

Reply:
To be sure I understand, may I ask kindly that you spell out FDs?

Reply:
File Descriptors (more accurately *os.File in Go)